numpy/core/fromnumeric.py

```diff
@@ -1223,6 +1239,13 @@ def argmin(a, axis=None, out=None):
     >>> np.argmin(b)  # Only the first occurrence is returned.
     0
 
+    >>> x = np.array([[4,2,3],[1,0,3]])
+    >>> np.take_along_axis(x, np.expand_dims(np.argmin(x, axis=-1), axis=-1), axis=-1)
```

Comment:
Possibly worth splitting these examples in two, either as:
```suggestion
    >>> index_array = np.argmin(x, axis=-1)
    >>> np.take_along_axis(x, np.expand_dims(index_array, axis=-1), axis=-1)
```
or
```suggestion
    >>> index_array = np.expand_dims(np.argmin(x, axis=-1), axis=-1)  # emulate keepdims=True
    >>> np.take_along_axis(x, axis=-1)
```